bindings/ruby/README.md

```diff
@@ -0,0 +1,101 @@
+# Regorusrb
+
+**Regorus** is
+
+  - *Rego*-*Rus(t)*  - A fast, light-weight [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/)
+   interpreter written in Rust.
+  - *Rigorous* - A rigorous enforcer of well-defined Rego semantics.
+
+Regorus can be used in Ruby by  Use the bundler CLI to add the gem from remote git source:
```

Comment:
Seems like part of the sentence got removed somehow.

Reply:
my mistake, updated now!